tests/dashbio_demos/app_alignment_viewer.py

```diff
@@ -4,6 +4,7 @@
 import dash_core_components as dcc
 from dash.dependencies import Input, Output
 import dash_bio
+import json
```

Comment:
You want this import in the first block (standard library imports).
Reference: https://www.python.org/dev/peps/pep-0008/#imports